### DISCUSSION TITLE
import piority of file.js and file.ts (same file title, different extensions)?

### DISCUSSION THREAD

**Discussion by ghost** [TypeStrong/ts-node#1287] [General]
In file `a.ts`:

```
import 'b'
```

when there are both files `b.ts` and the compile `b.js`, running `ts-node a.ts`, it import the file `a.js` as the priority.

I think this should be a bug. It should use the file with `.ts` extension in this case.

**Comment by cspotcode** [comment]
Check out the preferTsExts option.
